cmd/aspect/main.go

```diff
@@ -42,6 +42,16 @@ func main() {
 		fmt.Fprintln(os.Stderr, "Error:", err)
 		os.Exit(1)
 	}
+
+	setupErrors := pluginSystem.SetupPlugins().Errors()
```

Comment:
From the perspective of the main command, both `Configure` and `SetupPlugins` sound like they are doing the same thing. We can perhaps combine them all in a single method? Or maybe give different names to those methods so it's clearer what does what?

Reply:
`Configure` configures the plugin system. `SetupPlugins` configures the plugins themselves. I think they deserve to be separated. `ConfigurePluginSystem` and `ConfigurePlugins` might make the differences more clear.